src/main/kotlin/no/nav/familie/klage/infrastruktur/featuretoggle/FeatureToggleController.kt

```diff
@@ -13,7 +13,7 @@ import org.springframework.web.bind.annotation.RestController
 @Unprotected
 class FeatureToggleController(private val featureToggleService: FeatureToggleService) {
 
-    private val funksjonsbrytere: Set<Toggle> = setOf(Toggle.HENLEGG_FEILREGISTRERT_BEHANDLING, Toggle.TEST_ENVIRONMENT)
```

Comment:
meningen å fjerne begge?

Reply:
Ja, det settet var tomt i utgangspunktet. Lurer derfor på om vi kan fjerne controlleren ?

Reply:
Ser ut som at toggelen "henlegg feilregistrert behandling" brukes i backend, og at vi trenger controlleren siden frontend kaller på den. Hvis vi evt legger til toggles for frontend senere, så blir de med.